src/app/MessageDef/CommandDataIBs.cpp

```diff
@@ -14,7 +14,7 @@
  *    limitations under the License.
  */
 
-#include "InvokeRequests.h"
```

Comment:
This seems like an odd rename? Why not keep InvokeRequest? We have InvokeResponse.

Reply:
In IM spec, We do have InvokeResponseIB, but we don't have InvokeRequestIB, instead, it is called CommandDataIB, naturally its array we call CommandDataIBs....https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/data_model/Encoding-Specification.adoc#encoding-StatusIB

Reply:
Can we either change both, or neither of them? I think it makes sense to have some symmetry here..

Reply:
No.......CommandDataIB is part of invokeResponseIB....https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/data_model/Encoding-Specification.adoc#encoding-InvokeResponseIB, invokeResponseIB is part of InvokeResponseMessage...

At the same time, CommandDataIB Array is part of InvokeRequestMessage
https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/data_model/Encoding-Specification.adoc#encoding-InvokeResponseIB

Reply:
the best effort here is to keep InvokeRequests naming

Reply:
just updated PR to align with your first comment, thanks